src/hydrabus/hydrabus_mode_smartcard.c

```diff
@@ -63,6 +63,7 @@ static void init_proto_default(t_hydra_console *con)
 	proto->config.smartcard.dev_guardtime = 16;
 	proto->config.smartcard.dev_phase = 0;
 	proto->config.smartcard.dev_convention = DEV_CONVENTION_NORMAL;
+	proto->config.smartcard.dev_timeout = 10000;
```

Comment:
Why default timeout is so long (10000ms => 10s) ? (It will be better to be less than 1s especially as default value and usually any HW shall return data in less than 1s)

Reply:
I also found it huge but I kept the value used by `bsp_smartcard_read_u8()` in `src/drv/stm32cube/bsp_smartcard.c` (`SMARTCARDx_TIMEOUT_MAX`).